force-app/main/default/lwc/nksChatAuthenticationInfoV2/nksChatAuthenticationInfoV2.js

```diff
@@ -14,8 +14,6 @@ import CHAT_LOGIN_MSG_NO from '@salesforce/label/c.NKS_Chat_Login_Message_NO';
 import CHAT_LOGIN_MSG_EN from '@salesforce/label/c.NKS_Chat_Login_Message_EN';
 import CHAT_GETTING_AUTH_STATUS from '@salesforce/label/c.NKS_Chat_Getting_Authentication_Status';
 import CHAT_SENDING_AUTH_REQUEST from '@salesforce/label/c.NKS_Chat_Sending_Authentication_Request';
-import { subscribe as messageServiceSubscribe, APPLICATION_SCOPE, MessageContext } from 'lightning/messageService';
-import CHAT_MESSAGE_CHANNEL from '@salesforce/messageChannel/chatMessageChannel__c';
```

Comment:
Brukes denne messageChannelen noen andre steder, for hvis ikke burde vi vel også slette xml fila te den og?